classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_reader :label
+  attr_accessor :students
```

Comment:
- [ ] Dear @touredri, as it's required to be able to set the value of the `label` attribute, we need to have a getter and setter instead of just a getter, Please use `attr_accessor` instead of `attr_reader` for your `:label`